docs/matrix-org/privacy_notice.md

```diff
@@ -261,6 +261,15 @@ You can access all that we collect about you by using any compatible Matrix clie
 
 In unencrypted and encrypted rooms, users connecting to the matrix.org homeserver (directly or over federation) will be able to see messages and files according to the access permissions configuration of the relevant room. This data is stored in the format it was received on our servers, and can be viewed by Element engineers (employees and contractors) under the conditions outlined below.
 
+Rooms have different visibility settings which are determined by the room administrators. The history visibility possibilities are the following, by increasing order of openness (least open first):
+
+- `joined`: people need to join the room to see the history, and will only see the messages sent after they joined. This behaviour is similar to the experience of IRC on a bouncer, and is how all IRC channels are bridged to Matrix by default
```

Comment:
> and is how all IRC channels are bridged to Matrix by default

I'm super worried about this appearing in the privacy doc:

- This is true for one of our bridge implementations.
- This is only true for matrix.org's bridges.
- At the very least we should namedrop the instances this applies to. Otherwise,some rando old IRC bridge might start bridging stuff in world_readable and everyone gets up in arms again.